UI/pom.xml

```diff
@@ -96,7 +96,7 @@
               <goal>bower</goal>
             </goals>
             <configuration>
-              <arguments>install</arguments>
```

Comment:
@IkaroSales We have removed bower and PR has been submitted for the same. Hence the below change may not be required

Reply:
@IkaroSales @rvema - this is definitely not necessary. Just ensure that your user owns the files in this directory.